### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by SergioRZMasson**

### CODE REVIEW

File: README-review.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+cd ios
+pod update
+cd ..
+```
+
+(Android Emulator Only)
+
+In order to use Babylon React Native with the Android Emulator it is required that the amulator is configure with GLES3, use the following [instructions on how to set it up properly](docs/ANDROID_EMULATOR.md).
+
+
```
Comment (bghgary) [inline README-review.md:46]: I think we are missing a section where it's explained how developers can add JS code? I'm assuming we will recommend people to open App.tsx file with vscode?

File: README-review.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+## More information
+
+[How to create this project from scratch](CREATE.md)
+
+Additional Documentation
+------------------------
+
+* [BabylonJS website](https://www.babylonjs.com/)
+* [Babylon React Native](https://github.com/BabylonJS/BabylonReactNative)
+* [BabylonJS Playground](https://www.babylonjs-playground.com/)
```
Comment (bghgary) [inline README-review.md:66]: I'm not sure how useful these links are. The github one is useful, but what's the point of the other two?

Comment (SergioRZMasson) [reply README-review.md]: I agree, I will leave only the link for github, Playground will already be reference on the "Add you code" session.

File: README-review.md (hunk 1 above)
Comment (SergioRZMasson) [reply README-review.md:46]: Just added a "Add you code" session, with instruction on how to add custom scene setup.

File: README-review.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+
+## 1. Download 
+
+You can download source code archives from [GitHub](https://github.com/SergioRZMasson/BabylonReactNative-GetStarted) or use ```git``` to clone the repository.
+
+```
+> git clone https://github.com/SergioRZMasson/BabylonReactNative-GetStarted.git
+Cloning into 'BabylonReactNative-GetStarted'...
+```
```
[the remark this answers is not in the document]
Comment (SergioRZMasson) [reply README-review.md:21]: Added template support.

**Pull request merged**